arm-software/embedded/docs/atfe-dev-drop-builds.md

```diff
@@ -0,0 +1,48 @@
+# ATfE Dev Drops Builds
+
+## Overview
+
+ATfE Dev Drop builds provide the community with early access to upcoming
+versions of the product ahead of official releases.
+These builds are made available to support testing and early validation of new
+features.
+Please note that Dev Drop builds may be unstable, and could contain bugs. They
+are intended for evaluation and integration purposes only, and not recommended
+for production use.
+
+## Release Cycle
+
+Dev Drop builds are automatically generated nightly as part of the ATfE Nightly
+Build and Test workflow on GitHub Actions.
+Once all CI and CD stages complete successfully, a set of build artifacts is
+produced and uploaded.
+
+## Accessing Dev Drop Build Artifacts
+
+Dev Drop build artifacts are available through the GitHub Actions section of
+the repository **arm-toolchain**. To locate them:
+
+1. Navigate to the Actions tab on GitHub
+2. Select the workflow titled [“ATfE Nightly Build and Test”](https://github.com/arm/arm-toolchain/actions/workflows/atfe_nightly_build_and_test.yml)
```

Comment:
Since we're providing a direct link to the workflow, I think we don't need to suggest a separate step for going into the Actions tab.

```suggestion
1. Navigate to the [“ATfE Nightly Build and Test”](https://github.com/arm/arm-toolchain/actions/workflows/atfe_nightly_build_and_test.yml) Workflow in the Actions tab on GitHub
```